web/modules/custom/joinup_news/config/install/core.entity_view_display.node.news.default.yml

```diff
@@ -20,11 +20,29 @@ dependencies:
   module:
     - comment
     - ds
+    - field_group
     - file
     - link
-    - options
     - text
     - user
+third_party_settings:
```

Comment:
It is about adding entity_reference__listing_tile suggestion to entity reference fields (it adds top margin and border to these fields)